### ISSUE TITLE
ROIs block clicks to scatter plot points behind them

### ISSUE THREAD

**Issue by SpikeHarris** [pyqtgraph/pyqtgraph#1126]
### Short description
ROIs block clicks to scatter plot points behind them.

### Code to reproduce
```python
import pyqtgraph as pg

def aaa(points):
    print 'AAAAAAAAAA'

app = pg.mkQApp()
win = pg.QtGui.QMainWindow()

layout = pg.GraphicsLayoutWidget()
parent = pg.PlotItem()
scatter = pg.ScatterPlotItem(size=10, pen=None)
roi = pg.RectROI((-1, -1), (2, 2))

win.setCentralWidget(layout)
layout.addItem(parent)
parent.addItem(scatter)
parent.addItem(roi)

scatter.sigClicked.connect(aaa)

scatter.setData([0, 2], [0, 2])
parent.getViewBox().setRange(xRange=[-2, 3], yRange=[-2, 3])

win.show()
app.instance().exec_()
```

### Expected behavior
Clicking either of the two scatter points should print `AAAAAAAAAA`.

### Real behavior
Clicking the point behind the ROI prints nothing. Clicking the other point works as intended. Dragging the ROI until the mouse is on top of a point and then clicking also works as intended until the mouse is moved off the point.

### Tested environment(s)

 * PyQtGraph version: 0.10.0
 * Qt Python binding: PyQt4 4.11.4 Qt 4.8.7
 * Python version: 2.7
 * Operating system: Ubuntu 16.04
 * Installation method: conda

**Comment by ixjlyons** [comment]
I agree this isn't ideal, but I'm not sure what should/could be done about it. I could imagine we could make mouse interaction with the ROI only work when the mouse is near the border of the ROI, but I haven't really thought through how that'd work, especially for other ROI classes.
